config/puma.rb

```diff
@@ -1,65 +1,5 @@
 #!/usr/bin/env puma
 
-# The directory to operate out of.
-#
-# The default is the current directory.
-#
-# directory '/u/apps/lolcat'
-
-# Use a object or block as the rack application. This allows the
-# config file to be the application itself.
-#
-# app do |env|
-#   puts env
-#
-#   body = 'Hello, World!'
-#
-#   [200, { 'Content-Type' => 'text/plain', 'Content-Length' => body.length.to_s }, [body]]
-# end
-
-# Load “path” as a rackup file.
-#
-# The default is “config.ru”.
-#
-# rackup '/u/apps/lolcat/config.ru'
-
-# Set the environment in which the rack's app will run. The value must be a string.
-#
-# The default is “development”.
-#
-# environment 'production'
-
-# Daemonize the server into the background. Highly suggest that
-# this be combined with “pidfile” and “stdout_redirect”.
-#
-# The default is “false”.
-#
-# daemonize
-# daemonize false
-
-# Store the pid of the server in the file at “path”.
-#
-# pidfile '/u/apps/lolcat/tmp/pids/puma.pid'
-
-# Use “path” as the file to store the server info state. This is
-# used by “pumactl” to query and control the server.
-#
-# state_path '/u/apps/lolcat/tmp/pids/puma.state'
-state_path '/home/deploy/apps/kablammo/shared/sockets/puma.state'
-
-# Redirect STDOUT and STDERR to files specified. The 3rd parameter
-# (“append”) specifies whether the output is appended, the default is
-# “false”.
-#
-# stdout_redirect '/u/apps/lolcat/log/stdout', '/u/apps/lolcat/log/stderr'
-stdout_redirect '/home/deploy/apps/kablammo/shared/log/stdout.log', '/home/deploy/apps/kablammo/shared/log/stderr.log', true
```

Comment:
All these references to `/home` were causing puma to fail on macos. I removed them and it kept working, so I'm guessing that modern puma has reasonable defaults for these values?